components/private/ThumbTrack/src/MDCThumbTrack.m

```diff
@@ -1029,7 +1030,8 @@ - (void)updateTrackMask {
 
   if ((!self.enabled && _disabledTrackHasThumbGaps) ||
       ([self isValueAtMinimum] && _thumbIsHollowAtStart &&
-       !(_shouldDisplayDiscreteValueLabel && _numDiscreteValues > 0 && _isDraggingThumb))) {
+       !(!_continuousUpdateEvents && _shouldDisplayDiscreteValueLabel && _numDiscreteValues > 0 &&
```

Comment:
NB: This is a gnarly `if` statement, so if there's a way you can think of to break it up into sensible variables, that would be 💯

Reply:
Done.